SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Security Policy
+
+This project is still under development, the primary goal at this stage is to implement features but we also take security very seriously. This document defines the policy on how to report vulnerabilities and receive updates when patches to those are released.
+
+[Join Nervos Security Team!](mailto:careers@nervos.org)
+
+## Reporting a vulnerability
+
+All security bugs should be reported by sending email to [Nervos Security Team \<security@nervos.org>](mailto:security@nervos.org). Please encrypt your mail using GPG with the following public key.
```

Comment:
To report any security bugs, send an email to the Nervos Security Team. Please encrypt your email using GPG with the following public key: